---
source: test_programs/tests/execute.rs
expression: "&format!(\"success: {:?}\\nexit_code: {}\\n----- stdout -----\\n{}\\n----- stderr -----\\n{}\",\noutput.status.success(), output.status.code().unwrap_or(!0), stdout, stderr)"
---
success: false
exit_code: 1
----- stdout -----

----- stderr -----
error: Could not determine array length `-1`, encountered error: `The value `-1` cannot fit into `numeric u32` which has a maximum size of `4294967295``
   ┌─ compile_failure/arithmetic_generics_intermediate_underflow/src/main.nr:13:5
   │
13 │     push_zero(pop(array))
   │     ---------
   │

Aborting due to 1 previous error
